splinterd/api/static/openapi.yaml

```diff
@@ -1577,6 +1577,14 @@ paths:
         authenticate the user.
       parameters:
         - $ref: "#/components/parameters/protocol_version"
+        - name: redirect_url
+          in: query
+          description: |-
```

Comment:
Is the `-` here intentional?

Reply:
`|-` seems to be used interchangeably with just `|` throughout the doc, I was just following an example that used `-`